soil-gas/README.md

```diff
@@ -0,0 +1,44 @@
+Soil gas flux measurements have also been made at Raoul Island, Ngauruhoe, Tongariro, and Moutohorā Whale Island volcanoes. In these cases, pegs were not used to mark measurement points, so measurements could not be made at exactly the same positions each time. Instead, measurements are made in broadly the same areas, but subject to constraints such as access and time availability, which will vary from survey to survey. For this reason, making the data available through Tilde is not an option. Those data are available through this repository.
+
+## Data
+The location of each gas flux measurement is measured by a phone with a built-in GPS and recorded alongside the flux measured. The ground temperature at a depth of 10 cm is measured immediately after the soil gas measurement by inserting a thermocouple in the ground at the middle where the chamber was located.
+
+Measurement locations are provided in the [New Zealand Transverse Mercator 2000 (NZTM2000) grid](https://www.linz.govt.nz/guidance/geodetic-system/coordinate-systems-used-new-zealand/projections/new-zealand-transverse-mercator-2000-nztm2000), rather than latitude and longitude in the [World Geodetic System 1984 (WGS84)](https://www.linz.govt.nz/guidance/geodetic-system/coordinate-systems-used-new-zealand/geodetic-datums/world-geodetic-system-1984-wgs84) coordinate system that is commonly used for GeoNet spatial data.
+
+## Description of Files and Data Fields
+Each file in this repository contains the data collected from one survey of one volcanic area. Files are in CSV (comma-separated values) format.
+
+### Filenames
+A filename consists of three parts:
+- The volcano name, e.g "MoutohoraWhaleIsland". If both Te Reo and English names are commonly used for the volcano, the Te Reo name is followed by the English name. There are no spaces in the volcano name, and no macrons are used for Te Reo names.
+- The date the data were collected, e.g. "2025-01-29". This uses local time (NZST or NZDT). The date is separated from the volcano name by an underscore “\_”.
+- A “csv” suffix. This indicates a CSV file format. The suffix is separated from the date by a dot “.”.
+If measurements were carried out over more than one day, but are considered by the VMG  to represent a single dataset, they are provided as a single file without highlighting which observations were made on which day. However, the date part of the filename shows both measurement dates.
+
+### Data Fields
+| Field | Description |
+| --- | --- |
+| MeasurementNumber | A number assigned to a measurement location. Measurement numbers from one survey (data file) have no relation to measurement numbers from another survey (data file). |
+| NZTM-mE | New Zealand Transverse Mercator projection East coordinate, in units of metres. |
+| NZTM-mN | New Zealand Transverse Mercator projection North coordinate, in units of metres. |
+| CO2 | Carbon dioxide (CO<sub>2</sub>) flux, in units of g/m<sup>2</sup>/day (grams per square metre per day). |
```

Comment:
So CO2 is always measured but CH4 and H2S are optional? This might be stated explicitly for clarity, given the optionality of the others is stated.